wgpu-hal/Cargo.toml

```diff
@@ -104,6 +104,7 @@ gles = [
     "naga/glsl-out",
     "dep:arrayvec",
     "dep:bytemuck",
+    "dep:cfg-if",
```

Comment:
I notice that before this change, `cfg-if` is only a dev-dependency, never a library dependency. I assume there is no strong reason for that; I can rewrite the code to use only `#[cfg]` if desired.

Reply:
it's an absolutely tiny dependency and makes code look nicer here, don't mind it :)